LIPS/lip-26.md

```diff
@@ -0,0 +1,279 @@
+---
+lip: 26
+title: Community Staking Module
+status: Proposed
+author: Dmitry Gusakov, Sergey Khomutinin, Dmitry Chernukhin, Vladimir Gorkovenko
+discussions-to: https://research.lido.fi/t/community-staking-module/5917
+created: 2024-08-02
+updated: 2024-08-02
+---
+
+
+# LIP-26. Community Staking Module
+
+## Simple Summary
+
+This proposal introduces Community Staking Module (CSM) — the first permissionless staking module within the [Staking Router](https://research.lido.fi/t/lip-20-staking-router/3790) architecture. This module will not only allow for the permissionless entry to the Lido on Ethereum validator set. Still, it will also create a clear pathway for the existing solo and community stakers to join Lido on Ethereum validator set on privileged conditions thanks to the Early Adoption mechanics.
+
+The  [Community Staking Landscape](https://hackmd.io/@lido/Byp775Ay6) describes the initial motivation for CSM development. The [CSM Architecture](https://hackmd.io/@lido/rJMcGj0Ap) document provides a detailed description of the CSM design principles. This document also provides a detailed description of the technical implementation of CSM and related software, which should be considered an extension of the [CSM Architecture](https://hackmd.io/@lido/rJMcGj0Ap).
+
+## Motivation
+
+The fact that the Lido Validator set is permissioned makes it impossible for the community stakers to participate. Since community stakers participation is one of the top priorities of Ethereum, it is crucial to add a permissionless entry to the Lido on Ethereum Node Operator set and enfranchise solo-staker involvement in the protocol to stay aligned with Ethereum. At the same time, it is vital to maintain Lido on Ethereum protocol security, reliability, and capital efficiency on a level equal to or above the existing ones.
+
+Given the Staking Router architecture introduced in the [Lido V2](https://blog.lido.fi/introducing-lido-v2/) upgrade, it is proposed to implement a staking module that will add a permissionless entry to the Lido on Ethereum Node Operator set, enfranchise solo-staker participation in the protocol, and maintain Lido on Ethereum protocol security, reliability, and capital efficiency on a level equal to or above the existing ones.
```

Comment:
`security, reliability, and capital efficiency on a level equal to or above the existing ones.`
Amen. 

Do we really need to repeat it after each Paragraph? I mean, there are the same words at the end of the previous paragraph

Reply:
Fair. Will remove the duplicate

Reply:
Fixed

Reply:
Yeah, I believe it's a bit more for marketing activities rather then for technical specs :pray: